### PR TITLE
add telemetry

### PR DESCRIPTION
**Pull request by dmytronasyrov**
add telemetry

### CODE REVIEW

File: lib/uniris/application.ex [hunk 1]
```diff
@@ -43,7 +45,12 @@ defmodule Uniris.Application do
     ]
 
     opts = [strategy: :rest_for_one, name: Uniris.Supervisor]
-    Supervisor.start_link(Utils.configurable_children(children), opts)
+    res = Supervisor.start_link(Utils.configurable_children(children), opts)
+
+    :telemetry.execute([:uniris, :run_app, :success], %{"latency" => 1.2}, %{"hello" => "uniris"})
```
Comment (samuel-uniris) [inline lib/uniris/application.ex:50]: I think those measurements were here for tests . Maybe a cleanup ..

File: lib/telemetry/instrumenter.ex [hunk 2]
```diff
@@ -0,0 +1,20 @@
+defmodule Uniris.Telemetry.Instrumenter do
```
Comment (samuel-uniris) [inline lib/telemetry/instrumenter.ex:1]: I think we can leverage TelemetryMetrics here, with a given supervisor to support the right reporter and the list of metrics to support.

**Review by samuel-uniris — CHANGES_REQUESTED** [review]

File: lib/uniris/application.ex (hunk 1 above)
Comment (dmytronasyrov) [reply lib/uniris/application.ex:50]: You mentioned you want a simple example of telemetry usage. That's it.

Comment (samuel-uniris) [reply lib/uniris/application.ex:50]: I wanted a simple setup but ok will change later

File: lib/telemetry/instrumenter.ex (hunk 2 above)
Comment (dmytronasyrov) [reply lib/telemetry/instrumenter.ex:1]: Not sure exactly what you mean, but I think I did it in the correct fashion. I added a polling telemetry events example into the supervision tree. Take a look.

File: lib/uniris/application.ex (hunk 1 above)
Comment (dmytronasyrov) [reply lib/uniris/application.ex:51]: Let me clean these rows. We have a setup and it works as expected.

File: lib/telemetry/instrumenter.ex (hunk 2 above)
Comment (samuel-uniris) [reply lib/telemetry/instrumenter.ex:1]: Ok good now

**Review by samuel-uniris — APPROVED** [review]

**Pull request merged**